docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -371,13 +392,44 @@ protected function getAttachmentData(array $files) {
     if (!empty($files)) {
       foreach ($files as $fid) {
         $currentFile = File::load($fid);
-        $base64 = base64_encode(file_get_contents($currentFile->getFileUri()));
-        $fileContents[] = [
-          'content_type' => $currentFile->getMimeType(),
-          'filedata' => $base64,
-          'filename' => $currentFile->getFilename(),
-          'filesize' => (int) $currentFile->getSize(),
-        ];
+        $virusScanStatus = $currentFile->get('field_virus_scan_status')->value;
```

Comment:
I had to use getString() instead of value to get `field_virus_scan_status`.

Reply:
Hmm... `->value` works (note, not `->value()`), but `getString()` is fine as well.